RadixWallet/Features/SplashFeature/Splash.swift

```diff
@@ -91,7 +94,11 @@ public struct Splash: Sendable, FeatureReducer {
 					#endif
 					await send(.internal(.advancedLockStateLoaded(isEnabled: isEnabled)))
 				} else {
-					await send(.internal(.advancedLockStateLoaded(isEnabled: false)))
+					if !userDefaults.appLockMessageShown {
```

Comment:
Will this be a generic error message, that it applies for users with iOS 18 (with native App Lock support) and users with iOS 16/17 (that would need to use our custom app lock feature)?

Because I understood on Friday that it was meant only for iOS 18 users, in which case we need to add the `if #available` check to avoid situations as the following:

![Screenshot 2024-09-09 at 12 51 10](https://github.com/user-attachments/assets/0a5c6358-dd20-43e5-adbc-62869f68f99d)

Reply:
Also, do we want to have this on Splash view? Maybe it makes more sense to show it next time the user opens the Wallet after setting it up. 

Otherwise, this is the first message that a new user who just downloaded the Wallet on iOS 18 sees.

Reply:
>Because I understood on Friday that it was meant only for iOS 18 users, in which case we need to add the if #available check to avoid situations as the following:

You're right, an if #available check is needed. Will fix it.

>Also, do we want to have this on Splash view? Maybe it makes more sense to show it next time the user opens the Wallet after setting it up.
Otherwise, this is the first message that a new user who just downloaded the Wallet on iOS 18 sees.

Good point! In this case we can stick with the requirement to inform only users who had enabled Advanced Lock before iOS 18 update. The reason for displaying the message on the Splash view is that, at this step, users expect to authenticate with biometrics (like it used to be for them before iOS 18). Will check with Matt.